task/buildah-oci-ta/0.3/buildah-oci-ta.yaml

```diff
@@ -725,3 +754,36 @@ spec:
         requests:
           cpu: 100m
           memory: 256Mi
+  sidecars:
```

Comment:
Despite the functionality is behind a feature flag, it's still specific to PNC only.
I think we should keep the buildah task generic and create a dedicated pipeline for specific purposes (like FBC, Java, etc.)